File: server/src/main/java/com/fluffy/global/web/cookie/CookieProperties.java
package com.fluffy.global.web.cookie;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "auth.cookie")
public record CookieProperties(
        String accessTokenKey,
        boolean httpOnly,
        boolean secure,
        String domain,
        String path,
        int maxAge
) {
}

File: server/src/main/kotlin/com/fluffy/global/web/cookie/CookieProperties.kt
package com.fluffy.global.web.cookie

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "auth.cookie")
data class CookieProperties(
    val accessTokenKey: String,
    val httpOnly: Boolean,
    val secure: Boolean,
    val domain: String,
    val path: String,
    val maxAge: Long
)
